website/_includes/footer_custom.html

```diff
@@ -0,0 +1,31 @@
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!--
+Used from:
+https://github.com/just-the-docs/just-the-docs/blob/main/_includes/components/footer.html
+-->
+<footer>
+  <p><a href="#top" id="back-to-top">Back to top</a></p>
+
+  <p>
+    <a
+      href="https://github.com/carbon-language/carbon-lang/blob/trunk/{{ page.path }}"
+      id="edit-this-page"
+      >View on GitHub</a
+    >
+  </p>
+
+  <p class="text-small">
+    Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+    Exceptions. See <a href="/LICENSE">/LICENSE</a> for license information.
```

Comment:
Maybe a wording tweak and link to GitHub?

```suggestion
    Part of the Carbon Language project, licensed under the
    <a href="https://github.com/carbon-language/carbon-lang/blob/trunk/LICENSE">Apache License v2.0 with LLVM
    Exceptions<a>.
```